doc/source/generated/selections/base.txt

```diff
@@ -1,7 +1,7 @@
 ..
     Generated by gen_standard_selections.py
 
-==  ==  ==  ==  ==  ==  ==  ===
```

Comment:
It's not clear from your PR message as to why these are all changing?

Reply:
Not sure TBH. They change when I build the documentation, so I committed them since they are not ignored. They will likely be overwritten when the docs are build in CI?

Reply:
🤷🏽 maybe @lilyminium knows

Reply:
Ah yeah, sorry, I think I can't ignore them because the docs are built in CI? Is there a way to ignore files only from a user commit and not from a GH-Actions commit?

But yeah they get rebuilt.

Reply:
The changes are odd though. I think this means there's already txt files in the `develop` branch that we should probably remove. The ones in gh-pages (at least rendered at the https://userguide.mdanalysis.org/stable/units.html) look up to date with Rocco's, so that's not too bad.

Reply:
Ok, so it is safe to leave them?

Reply:
Yeah -- or you could remove them completely, doesn't make much sense to keep txt files in the repo. Not much difference either way. Maybe you don't want your path commented in the tables though?

Reply:
Good point. So I can remove all files in `generated`?

Reply:
I think so yeah

Reply:
How can I re-generate these files? If I run `make html` after removing them I get a bunch of errors:
```
[...]
/Users/rmeli/Documents/git/software/UserGuide/doc/source/topology_system.rst:152: CRITICAL: Problems with "include" directive path:
InputError: [Errno 2] No such file or directory: 'source/generated/topology/defaults.txt'.
/Users/rmeli/Documents/git/software/UserGuide/doc/source/topology_system.rst:287: CRITICAL: Problems with "include" directive path:
InputError: [Errno 2] No such file or directory: 'source/generated/topology/groupmethods.txt'.
[...]
```

Reply:
I'm going to say at this point, let's raise an issue and deal with it later? If @lilyminium says there's not much difference either way then it should be fine here for a bit.